### PR TITLE
Add backward compatibility for old block id mapping file

### PR DESCRIPTION
**Pull request by petterarvidsson**

### CODE REVIEW

**Comment by petterarvidsson** [comment]
@nsg This will be run every time the server starts. If the old file is found new files will be created and then it will be deleted. If it's not found server continues the start-up sequence as usual. If anything fails, the start-up sequence is aborted to protect the data server data.

File: src/main/scala/konstructs/main.scala [hunk 1]
```diff
@@ -1,13 +1,71 @@
...
+
+  /*
+   * This "migration" only works if disk storage on default location was used
+   */
+  def handleOldUuidDB() {
+    val gson = GsonDefault.getDefaultGson
+    try {
+      val fileName = s"meta/org/konstructs/block-manager/position-mapping.json"
+      val reader = new FileReader(fileName)
+      val mapping: java.util.Map[String, UUID] = gson.fromJson(reader, TypeOfPositionMapping)
```
Comment (nsg) [inline src/main/scala/konstructs/main.scala]: What happens if position-mapping.json is not there?

Comment (petterarvidsson) [inline src/main/scala/konstructs/main.scala]: Then you end up in the catch clause, skipping the process. Either it's a new server or it was already migrated.

Comment (nsg) [inline src/main/scala/konstructs/main.scala]: Ok, sounds good!

Comment (petterarvidsson) [inline src/main/scala/konstructs/main.scala]: It's not the most elegant code ever written :-) Thanks for checking it out!

**Pull request merged**